tests/agent_unittests/test_import_hook.py

```diff
@@ -0,0 +1,43 @@
+from pytest import raises
+
+
+def test_import_hook_finder():
+    """
+    This asserts the behavior of ImportHookFinder.find_module. It behaves
+    differently depending on whether or not the module it is looking for
+    exists, has been registered with an import hook, and across different
+    python versions.
+    """
+    from newrelic.api.import_hook import ImportHookFinder, register_import_hook
+    import sys
```

Comment:
(suggestion) These test local imports can probably be moved to the module scope (outside of the test)